docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
+
+- Provide feedback on proposals or ideas
+- Discuss possible use-cases or feature requests
+- Make suggestions for non-technical contributions, including events, writing, or business models
+- Answer questions about the protocol, services and products.
```

Comment:
I would add either periods or commas to each bullet point:

```suggestion
- Provide feedback on proposals or ideas.
- Discuss possible use-cases or feature requests.
- Make suggestions for non-technical contributions, including events, writing, or business models.
- Answer questions about the protocol, services and products.
```

Reply:
Majella said no to this style and said at the end of all bullet points a fullstop is fine.